android/kotlin-davinci/README.md

```diff
@@ -9,7 +9,23 @@
 
 Ping provides these Android samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
 
-This repository contains an example Android project written in Kotlin making use of the SDK. The sample supports the OOTB DaVinci Login flow.
+This repository contains an example Android project written in Kotlin making use of the SDK. The sample supports the OOTB DaVinci Login flow and OIDC Centralized Login.
+
+# Introduction
+
+This sample application demonstrates how to integrate the ForgeRock Android SDK into a basic application. The sample app includes examples of the following:
+
+- OOTB DaVinci Login flow
+
+- Centralized Login: Implements a secure, browser-based authentication flow using OAuth 2.0 and PKCE to connect to a DaVinci user journey.
```

Comment:
We use Journey for AIC, the terms DaVinci user journey is confusing.
`Centralized Login: Implements a secure, browser-based authentication flow using OAuth 2.0 and PKCE`